plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/aps/system/services/resource/model/ResourceInterface.java

```diff
@@ -217,6 +217,15 @@
      */
     public void saveResourceInstances(ResourceDataBean bean) throws ApsSystemException;
 
+    /**
+     * Ricava ed salva tutte le istanze associate ad una risorsa, valorizzando
+     * quest'ultima con i dati delle istanze ricavate.
+     *
+     * @param bean L'oggetto detentore dei dati della risorsa da inserire.
+     * @throws ApsSystemException In caso di eccezioni.
+     */
+    public void saveResourceInstances(ResourceDataBean bean, List<String> ignoreMetadataKeys) throws ApsSystemException;
```

Comment:
Give that the method is overloaded lets update the comment to English and note the difference between the two.